enterprise/replay/dumpsloader.go

```diff
@@ -258,6 +271,12 @@ func (handle *dumpsLoaderHandleT) Setup(ctx context.Context, db *jobsdb.HandleT,
 	if err != nil {
 		panic("invalid start time format provided")
 	}
+
+	tmpDirPath, err := misc.CreateTMPDIR()
```

Comment:
I was trying to understand if tmp dir is persist across pod restarts, then I saw we are using it for badger db. In prod we setting it in `RUDDER_TMPDIR=/data/rudderstack`. 

TL;DR: TMPDIR is a confusing name